autogen/main/dns.tf.tmpl

```diff
@@ -68,6 +68,12 @@ EOF
   ]
 }
 
+# Fix the name typo in the previous ConfigMap creation call
+moved = {
```

Comment:
I think their might be a typo in the terraform docs, pretty sure this is a block:

```suggestion
moved {
```

Reply:
Updated on our end.
Funny enough, their source code has the right reference: https://github.com/hashicorp/terraform/blob/a384d2daa7fa7dfcf891c634c26de4a57ee4c963/website/docs/language/moved.mdx#complete-configuration.
But the published website has `moved = {`.

Reply:
https://developer.hashicorp.com/terraform/language/moved seems to have the right syntax.

Reply:
Ah, we were looking at _v1.3_ page, while the latest is _v1.9_!

Reply:
Good catch! - Guess it was fixed later, but they forgot to update across all the doc versions.

Reply:
FYI: This module currently requires TF >=1.3, so we typically reference the v1.3 docs to ensure we aren't including a feature which requires a later version.